### ISSUE TITLE
Svg Path stroke color should not fixed to '#000' instead of 'currentColor'

### ISSUE THREAD

**Issue by zhangriyueming** [grommet/grommet-icons#171]
Svg Path stroke color should not fixed to '#000' instead of 'currentColor'

**Comment by ShimiSun** [comment]
Can you please explain what do you mean by currentColor and how you are trying to use it?

**Comment by DavidJFelix** [comment]
@ShimiSun I found this issue wanting the same thing. I'm trying to use the icons with `react-icons` and I was unable to change the color of the icons like I'm able to with other kids like font-awesome and feather. The `currentColor` ask is that rather than define the stroke as black, it be set to the SVG label [currentColor](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/color). This means that by default in HTML, it will be black, but if a user sets a font color in a parent HTML element, the SVG will use that color for the stroke color. This allows people to easily style the icons for whatever they're doing.

**Comment by aravindanve** [comment]
@DavidJFelix you are right. If you are looking for a temporary solution, overriding the stroke using css worked for me. But the caveat is you will have to add the class `my-grommet-icon` to your icon component:

```css
.my-grommet-icon path {
  stroke: currentColor;
}
```

Or if you are using styled-system, theme-ui or rebass, you could do this:

```js
const GrIconWrapper = ({ icon: Icon, sx, ...props }) => (
  <Box
    {...props}
    as={Icon}
    sx={{
      ...sx,
      path: {
        stroke: 'currentColor',
      },
    }}
  />
);

// and use it like
<GrIconWrapper icon={GrForwardTen} />
```

**Comment by TomasB** [comment]
@ShimiSun It's a simple search and replace across all files. Do you need help with this? Should take only minutes to fix to make them usable.

**Comment by ShimiSun** [comment]
> @ShimiSun It's a simple search and replace across all files. Do you need help with this? Should take only minutes to fix to make them usable.

Go for it buddy, thanks!

**Comment by TomasB** [comment]
I'm resetting both 'fill' and 'stroke' to be 'currentColor' so that the values can be inherited.

**Comment by ShimiSun** [comment]
@TomasB Not sure what you are asking, would you like to file a contribution with the suggested fix you have in mind?
